someip2val/docker/vsomeip/examples/notify-sample.cpp

```diff
@@ -0,0 +1,305 @@
+// Copyright (C) 2014-2017 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
We need an Eclipse Apache header here if we argue the example is rather documentation or we need to do an CQ for MPL if we consider this a modified someip ocde

Reply:
I've removed modified examples, they had some dumps added, nothing important